icons/swatch-book.svg

```diff
@@ -0,0 +1,18 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M2 8h8" />
+  <path d="M2 14h8" />
+  <path d="M10 18a4 4 0 0 1-8 0V4a2 2 0 0 1 2-2h4a2 2 0 0 1 2 2Z" />
+  <path d="m10 8.3 3.1-3.1c.8-.8 2-.8 2.8 0L18.7 8c.8.8.8 2 0 2.8l-9.9 9.9" />
+  <path d="M15.7 14H20a2 2 0 0 1 2 2v4a2 2 0 0 1-2 2H6" />
+  <path d="M6 18h.01" />
```

Comment:
I agree with @jguddas, let's keep it simple.
```suggestion
  <path d="M11 17a4 4 0 0 1-8 0V5a2 2 0 0 1 2-2h4a2 2 0 0 1 2 2Z" />
  <path d="M16.7 13H19a2 2 0 0 1 2 2v4a2 2 0 0 1-2 2H7" />
  <path d="M 7 17h0.01" />
  <path d="m11 8 2.3-2.3a2.4 2.4 0 0 1 3.404.004L18.6 7.6a2.4 2.4 0 0 1 .026 3.434L9.9 19.8" />
```

It's also maybe a bit too large and definitely not optimized enough:
![image](https://github.com/lucide-icons/lucide/assets/17746067/36bfd44f-6393-448d-9c2c-f26af6043ae8)
![image](https://github.com/lucide-icons/lucide/assets/17746067/376e2e75-2004-42d1-88c9-6718e7dd4bb2)